modules/Makefile.circleci

```diff
@@ -45,6 +45,8 @@ circle\:tag-latest:
 	@$(SELF) docker:login
 	@echo "INFO: Tagging latest"
 	@$(SELF) DOCKER_TAG=latest docker:push
+	@git tag --force "$(CIRCLE_BRANCH)-docker-latest"
+	@git push origin --force "tags/$(CIRCLE_BRANCH)-docker-latest"
```

Comment:
I think this is ok for the moment, but there's a little bit of a dependency ordering issue (in some of the service repos anyway). The `circle:deploy-kubernetes` target is actually the one that applies and pushes the githash docker tag we need to do record keeping on. In practice most repos do tag-latest and deploy-kubernetes together but maybe we `circle:tag-latest` ought to depend on `circle:tag` instead?